modules/web/src/app/shared/components/cluster-summary/template.html

```diff
@@ -329,7 +329,7 @@ <h4>IPv6</h4>
           <div value>{{cluster.credential}}</div>
         </km-property>
 
-        <ng-container *ngIf="!cluster.credential">
+        <ng-container *ngIf="displaySettings()">
```

Comment:
Wouldn’t it be easier to just check if there’s no preset (!cluster.credential), since all the properties inside the ng-container are disabled when a preset is selected?